### ISSUE TITLE
Error in the first batch file

### ISSUE THREAD

**Issue by atrugiel69** [tool-maker/VPN_just_for_torrents#1]
Hi! I'm trying to use your batch files.

I launch the first batch before connecting to the VPN, through AirVPN Eddie program events feature. The batch is fully executed before the VPN is connected.

I get a:

'then' is not recognized as an internal or external command, operable program or batch file."

error message in the commandline window. I suppose it refers to line 60 in your batch.

I don't think this is stopping the batch from doing what it is supposed to do.
I check my IP on ipleak.net and everything is correct. I can use ForceBindIP to launch programs to work inside the VPN or I can use qBittorrent feature to bind the client to the VPN adapter. I checked and all of this works.

**Comment by tool-maker** [comment]
I cannot reproduce this message. 
I just tried copying the content of the script back into a new .bat file and ran that. No error message.

**Comment by atrugiel69** [comment]
Compared them, line by line with WinMerge, and the batch I'm executing is identical to the one you posted here.

Edit: while checking stuff I noticed you updated your message above. Checking now if you changed that code... you did. I'll update and report back.
And yeah, I disabled IPv6 in the VPN (and I think even in the normal adapter). I remember people stating it was a security risk for torrenting. Do I remember wrongly?
